client/lib/analytics/page-view-tracker/index.jsx

```diff
@@ -68,25 +68,39 @@ export class PageViewTracker extends React.Component {
 			path,
 			recorder = noop,
 			hasSelectedSiteLoaded,
+			isSelectedSiteNotRequested,
 			title,
-			properties,
 		} = this.props;
 
 		debug( `Queuing Page View: "${ title }" at "${ path }" with ${ delay }ms delay` );
 
-		if ( ! hasSelectedSiteLoaded || this.state.timer ) {
+		if ( this.state.timer ) {
```

Comment:
Not the most concise way to do it, but I believe it's easier to read.

Reply:
@monsieur-z I thought I understood the diff but your comment confuses me. can you share a bit more context, specifically:

 - _what_ is not the most concise way
 - to do _what_
 - _how_ is _what_ easier to read? to _whom_?

if I were to wager a guess I'm thinking you are trying to explain why you split the combined conditional into separate clauses?